deps/build.jl

```diff
@@ -1,22 +1,14 @@
 # MadNLP.jl
 # Created by Sungho Shin (sungho.shin@wisc.edu)
 
-
-Sys.iswindows() && error("Windows is currently not supported.")
+blasvendor=(haskey(ENV,"MADNLP_BLAS") && ENV["MADNLP_BLAS"]=="openblas") ? :openblas : :mkl
```

Comment:
I guess this should be `:openblas` for anything other than `Sys.ARCH ==  :x86_64`